scripts/known_good.json

```diff
@@ -64,7 +64,7 @@
             "sub_dir": "valijson",
             "build_dir": "valijson/build",
             "install_dir": "valijson/build/install",
-            "commit": "v1.0"
+            "commit": "v1.0.1"
```

Comment:
Please also update the version of valijson in VulkanTools. I'd like them to be consistent to avoid any confusion when we build the SDK.

Reply:
Ok!